mesonbuild/dependencies/boost.py

```diff
@@ -679,11 +682,6 @@ def detect_roots(self) -> None:
         else:
             tmp = []  # type: T.List[Path]
 
-            # Homebrew
-            brew_boost = Path('/usr/local/Cellar/boost')
-            if brew_boost.is_dir():
-                tmp += [x for x in brew_boost.iterdir()]
-
```

Comment:
What is the reason behind removing this?

Reply:
On homebrew systems, boost should be detected in `/usr/local/lib` instead of the cellar. Looking in the cellar made sense before when the logic couldn't follow symlinks. But since this patch adds support for following symlinks, it now finds boost in /usr/local/lib like it should. In addition, event if it somehow got to checking the cellar it wouldn't find boost_python because it is in a different homebrew formula.